roles/lib_openshift/src/lib/pvc.py

```diff
@@ -51,13 +61,29 @@ class PersistentVolumeClaim(Yedit):
     volume_name_path = "spec.volumeName"
     bound_path = "status.phase"
     kind = 'PersistentVolumeClaim'
+    selector_path = "spec.selector.matchLabels"
+    storage_class_name_path = "spec.storageClassName"
 
     def __init__(self, content):
         '''RoleBinding constructor'''
```

Comment:
If you wouldn't mind renaming this to PersistentVolumeClaim in the doc.  Seems like a copy paste from before.